lib/plenario2_web/controllers/data_set_field_controller.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Plenario2Web.DataSetFieldController do
+  alias Plenario2.Actions.{DataSetFieldActions, MetaActions}
+  alias Plenario2.Changesets.DataSetFieldChangesets
+  alias Plenario2.Repo
+  alias Plenario2.Schemas.DataSetField
+  use Plenario2Web, :controller
+
+  def index(conn, params) do
```

Comment:
For all of these controllers, I prefer seeing long form maps:

```elixir
def index(conn, %{"slug" => slug}) do ...
```

If you still want to capture the rest, you can do `%{"whatever" => variable} = params` and access keywords off the _params_ variable.

Also, _data set slug_ is too verbose -- just make it _slug_.